### DISCUSSION TITLE
Linking to other assets from details view

### DISCUSSION THREAD

**Discussion by dmosesson** [dagster-io/dagster#21078] [Q&A] [answered]
Is there a good way to link to other assets that are related, but logically rather than by processing flow (i.e. the following assets have FK relationships to this asset)?

**Comment by garethbrickman** [comment]
Could you elaborate what you mean by "link to"? You could consider using[ Tags](https://docs.dagster.io/concepts/metadata-tags/tags#tags) for this

**Reply by dmosesson** [reply]
I think the example of foreign keys is probably the simplest. If an asset is a postgres table, and it holds foreign keys for other tables that are also dagster assets, what is the right way to illustrate that in the UI so that the analysts don't have see all the relationships in PGAdmin?

**Reply by garethbrickman** [reply]
Ah gotcha, sounds like [Asset metadata](https://docs.dagster.io/concepts/metadata-tags/asset-metadata#asset-metadata), and specifically [definition metadata](https://docs.dagster.io/concepts/metadata-tags/asset-metadata#definition-metadata), would be a way to make that info easily accessible to them in the Dagster UI.

**Reply by dmosesson** [reply]
I understand how to add text to the description, but is there a way to hyperlink to the other assets?

**Reply by garethbrickman** [reply, marked as answer]
The [MetadataValue class](https://docs.dagster.io/_apidocs/ops#dagster.MetadataValue) has the capability to set hyperlinks via MetadataValue.url however these would have to be hard coded. 

```python
from dagster import asset, MetadataValue

@asset(metadata={
    "url": MetadataValue.url("http://example.com")
})
def my_asset():
    # Asset computation logic
    pass
```

**Reply by dmosesson** [reply]
That works, thanks
